### PR TITLE
feat(academy): add part 1.4

### PR DESCRIPTION
**Pull request by MathieuConte**
Add content of part 1.4 "Protocol concepts" of the Academy.

Ready to review ! 

(Minor change upcoming : add new illustrations)

## Summary by CodeRabbit

- **Documentation**
    - Expanded the Academy section with detailed explanations of the protocol's core principles such as sovereignty, decentralization, interoperability, governance, and scalability.
    - Highlighted the protocol's use of IBC for interoperability and Cosmos technology for enhanced scalability.

### CODE REVIEW

File: docs/academy/protocol-concepts.md [hunk 1]
```diff
@@ -3,3 +3,210 @@ sidebar_class_name: hidden
...
+
+<u>Availability and Sustainability:</u> OKP4's decentralized infrastructure provides an uninterrupted settlement layer crucial for ensuring constant access to digital resources. Additionally, the system's decentralized nature guarantees its sustainability, making it viable despite disruptions or technological advancements. As a result, digital resource sharing can be sustained in the long run.
+
+### Decentralized Orchestration
+
+As an decentralized orchestration layer for off-chain resources, OKP4 must ensure two functionality in a decentralized manner :
```
Comment (ccamel) [inline docs/academy/protocol-concepts.md:45]:
```suggestion
As a decentralized orchestration layer for off-chain resources, OKP4 must ensure two functionalities in a decentralized manner:
```

File: docs/academy/protocol-concepts.md [hunk 2]
```diff
@@ -3,3 +3,210 @@ sidebar_class_name: hidden
...
+2. Pactum oversees the entire execution flow of all resources.
+3. Finally, the Pactum smart contract applies retribution rules, ensuring the distribution of rewards or compensation in accordance with defined retribution rules. In practical terms, Pactum functions as a decentralized regulator, coordinating the orchestration of digital resources and managing instances of error.
+
+## INTEROPERABILITY
+
+Interoperability is a critical aspect of the OKP4 protocol, which enables seamless interactions among diverse systems, resources, and digital services. In this context, interoperability is paramount as the OKP4 protocol aims to orchestrate and share digital resources. The protocol adopts a technologically agnostic approach, enabling transparent interaction among resources, technologies, and ecosystems, bridging the gap between Web 2.0 and Web 3.0 environments. Interoperability is key to promoting smooth and effective collaboration, as it eliminates technical barriers that may hinder the sharing and utilization of digital resources. OKP4 facilitates efficient and effective collaboration by enabling interoperability, thereby enhancing the value and utility of digital resources.
```
Comment (ccamel) [inline docs/academy/protocol-concepts.md:103]:
```suggestion
Interoperability, a critical aspect of the OKP4 protocol, enables seamless interactions among diverse systems, resources, and digital services. In this context, interoperability is paramount as the OKP4 protocol aims to orchestrate and share digital resources. The protocol adopts a technologically agnostic approach, enabling transparent interaction among resources, technologies, and ecosystems, bridging the gap between Web 2.0 and Web 3.0 environments. Interoperability is key to promoting smooth and effective collaboration, as it eliminates technical barriers that may hinder the sharing and utilization of digital resources. OKP4 facilitates efficient and effective collaboration by enabling interoperability, thereby enhancing the value and utility of digital resources.
```

File: docs/academy/protocol-concepts.md [hunk 3]
```diff
@@ -3,3 +3,210 @@ sidebar_class_name: hidden
...
+
+As an orchestration layer for off-chain resources, OKP4 enables users to connect and leverage Anything-as-a-service to share anythings in any conditions. All these shared resources and rules shape the Dataverse, an ever-expanding universe of datasets, algorithms and all other resources referenced and interconnected in the blockchain.
```
Comment (ccamel) [inline docs/academy/protocol-concepts.md:111]:
```suggestion
As an orchestration layer for off-chain resources, OKP4 enables users to connect and leverage Anything-as-a-service to share anythings in any conditions. All these shared resources and rules shape the Dataverse, an ever-expanding universe of datasets, algorithms, and all other resources referenced and interconnected in the blockchain.
```

File: docs/academy/protocol-concepts.md [hunk 4]
```diff
@@ -3,3 +3,210 @@ sidebar_class_name: hidden
...
+
+### Protocol Governance
+
+Protocol governance in OKP4 is built on the Cosmos stack's foundations, allowing participants to propose and vote on fundamental protocol changes. This decentralized approach ensures continuous scalability and adaptation to technological advancements. It results in transparent, consensus-based updates, strengthening participants' trust in the stability and future of the protocol.
+
+### Resource Governance
+
+At the resource level, governance is expressed through consent rules, strictly recorded and enforced on-chain. Resource providers define these rules, determining how their data or services can be used. This approach gives resource providers precise control over how their assets are shared, reinforcing trust and encouraging open adoption of digital resources.
+
+Here are some examples of rules that providers can customize:
```
Comment (ccamel) [inline docs/academy/protocol-concepts.md:161]:
```suggestion
Here are examples of rules that can be defined for a resource:
```

**Review by ccamel — CHANGES_REQUESTED** [review]
Some minor corrections.

It would also be appropriate to use the same case for all titles, conventionally capitalizing them. For example, "SOVEREIGNTY" should be changed to "Sovereignty".

**Review by MdechampG — APPROVED** [review]
Thank you Mathieu for this job ! 
I agree with Chris for the titles. Otherwise it looks very well to me. 
I just wonder if it's not too long : should we separate into different pages ? But Let's keep it like that for now and wait for the first feedbacks. 
We could also add some blocks at the beginning of the page to be redirected to the concept the reader is interested in. But maybe it's redondant with the lateral menu on the right 🤔

**Review by ccamel — APPROVED** [review]
Super! Thanks :+1:

**Pull request merged**